solutions/nx-monorepo/nx.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "npmScope": "with-nx",
+  "affected": {
+    "defaultBase": "main"
+  },
+  "implicitDependencies": {
+    "package.json": {
+      "dependencies": "*",
+      "devDependencies": "*"
+    },
+    ".eslintrc.json": "*"
+  },
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": ["build", "lint", "test", "e2e"]
+      }
+    },
+    "cloud": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": ["build", "lint", "test", "e2e"],
+        "accessToken": "YOUR ACCESS TOKEN HERE"
```

Comment:
Is this token private? if that's the case then does Nx support this being an environment variable/secret?

Reply:
Good call. It _can_ be, often times a read only token will be included here which is safe to publish. But there is an env variable that can be set that takes precedence over this setting - so I'll go that route and include that in a `.env.example`.

This also isn't required, you can just use local caching (the default in this example), but I wanted to include this here because when deploying on Vercel the _version_ of `nx-cloud` that is used matters ([details here](https://github.com/nrwl/nx/issues/10256#issuecomment-1233292721)) so I wanted to have a clear example to point people too.